packages/realm-react/src/__tests__/useQueryRender.test.tsx

```diff
@@ -405,7 +405,7 @@ describe.each`
         await new Promise((resolve) => setTimeout(resolve, 10));
         const id = i;
         testRealm.write(() => {
-          testRealm.create(Item, { id, name: `${id}` }, Realm.UpdateMode.Modified);
+          return new Item(testRealm, { id, name: `${id}` });
```

Comment:
Would it make sense to test both using `new` and `create` you think?

Reply:
That should be tested in the integration tests.  I'm only concerned about the hooks in these tests, so i'll just stick to our current best practices (which is to use `create`

Reply:
🙂